### ISSUE TITLE
Close button for home page banner is not accessible with keyboard navigation

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#13251]
**Setup**
- Rancher version: `master`
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

Related to https://github.com/rancher/dashboard/issues/12776

Based on comment https://github.com/rancher/dashboard/issues/12776#issuecomment-2620335461, there an issue where the X in the home page banner is not accessible via keyboard navigation because it's only displayed on hover. It needs to be keyboard accessible.

**To Reproduce**

Hover with mouse on banner image to display the close button (it will appear - it only appears on this scenario)

**Result**
X to close home banner is not accessible via keyboard navigation

**Expected Result**

We should be able to close the home page banner with keyboard navigation

**Screenshots**

**Additional context**

Proposed solution/fix: https://github.com/rancher/dashboard/pull/13228#issuecomment-2626795125

solution 2: https://github.com/rancher/dashboard/pull/13228#issuecomment-2625918107

**Comment by yonasberhe23** [comment]
e2e test adequately covers this update. No manual validation is required. Closing this ticket.
